uint16 seq
uint16 pos_x
uint16 pos_y
uint16 size_x
uint16 size_y